app/controllers/admin/users_controller.rb

```diff
@@ -78,7 +78,7 @@ def create
         flash[:success] = 'User has been created! An invite has been sent'
         redirect_to admin_users_path
       else
-        Rails.logger.debug @user.errors.full_messages
+        @partners = collect_partners
```

Comment:
I expected this to be `@partner` not a list of partners for the new view / form. What am I missing here?

Reply:
This variable is set up so the form view has a list of partners with some of those partners pre-selected. I think it is used when creating a partner: it allows us to have a link that takes the admin to the "New user" form with the just-created partner pre-selected.